experimental/dds/tree2/src/feature-libraries/deltaUtils.ts

```diff
@@ -78,12 +78,20 @@ export function mapMark<TIn, TOut>(
 			};
 		}
 		case Delta.MarkType.MoveOut: {
-			return {
-				type: Delta.MarkType.MoveOut,
-				count: mark.count,
-				moveId: mark.moveId,
-				...mapModifications(mark, func),
-			};
+			return mark.isDelete
```

Comment:
I recommend creating a local witout the `isDelete` populated first, and then doing:
```
if (mark.isDelete) {
	delta.isDelete = true;
}
```
It avoids the code duplication.